python/lsst/ts/phosim/CloseLoopTask.py

```diff
@@ -1013,7 +1194,15 @@ def runImg(
                 pathSkyFile=pathSkyFile,
             )
 
-        self.phosimCmpt.tele.setInstName(camType)
+        if inst == "lsst":
+            # Append equal weights for CWFS fields to OFC data
+            self.phosimCmpt.tele.setInstName(camType, defocalDist=0.0)
+            self.ofcCalc.ofc_data.normalized_image_quality_weight = np.append(
+                self.ofcCalc.ofc_data.normalized_image_quality_weight,
+                [0.25, 0.25, 0.25, 0.25],
```

Comment:
Please leave a comment to explain this magic number: [0.25, 0.25, 0.25, 0.25]. In the future, we may want to put this into the configuration file or use a new algorithm to adjust the weighting in the run time.